propagators/README.md

```diff
@@ -22,8 +22,8 @@ api.propagation.setGlobalPropagator(new GrpcCensusPropagator());
 
 ## Useful links
 
-* For more information on OpenTelemetry, visit: <https://opentelemetry.io/>
-* For more about OpenTelemetry JavaScript: <https://github.com/open-telemetry/opentelemetry-js>
+- For more information on OpenTelemetry, visit: <https://opentelemetry.io/>
+- For more about OpenTelemetry JavaScript: <https://github.com/open-telemetry/opentelemetry-js>
```

Comment:
nit: It would clarify this PR to not have the bullet changes included. Not a big problem, however.

Reply:
The issue is that I created the `Supported Versions` section in the READMEs, and was using common format of dash. and then the linter said I was using mixed unordered list format in the same markdown.

I can split it up into a separate PR if you prefer, was trying to save time by just doing it all at once after spotted.

Reply:
I extracted this change into #2205 

Will rebase this PR once it's merged to focus it only on supported versions documentation changes and nothing else

Reply:
Thank you for separating that change out!

Reply:
#2205 is merged and this PR is rebased.

closing this thread